.stylelintrc.json

```diff
@@ -31,6 +31,7 @@
     "selector-pseudo-element-no-unknown": true,
     "selector-type-no-unknown": true,
     "string-no-newline": true,
-    "unit-no-unknown": true
+    "unit-no-unknown": true,
+    "selector-class-pattern": "^([a-zA-Z][a-zA-Z0-9]*)(-[a-zA-Z0-9]+)*$"
```

Comment:
Probably it will be better to leave it as it was, and just add `/* stylelint-disable selector-class-pattern */` in this single place. Optionally, disable it all.